Cargo.toml

```diff
@@ -45,6 +45,9 @@ version = "0.2"
 name = "rustpython"
 path = "src/main.rs"
 
+[profile.dev.package."*"]
+opt-level = 3
```

Comment:
Wouldn't this slow down the `dev` build?

Reply:
Speaking for myself, but I always build with `--release`. I think that dev builds (with the default opt-level 0, at least) are just too slow, especially with the reports of defaulting with dev binaries because maybe calls don't get inlined or something.

Reply:
Though maybe opt-level 2 would be better for debug builds, since 3 makes there no difference between debug/release

Reply:
Yes, it will slow building dependencies (which rarely build again) but boost running speed.
For the building speed, dependencies are usually the same and we only rebuild rustpython part. So only the first build will be slowed down but we will always get the benefits of running speed.
Because we usually only need to debug RustPython part, I think this will only cause speedup of running speed of dependencies and (mostly) nothing to lose (Though this option must be edited when we also need to debug dependencies - which is not very ideal)

I don't see any reason not to use the same option for debug/release build. They are not the part of RustPython so they must be blackbox for usual use case. So using same option is actually better because users anyway will use them only with release build. But it is considerable If build speed of opt-level 2 is a lot better than 3.

Reply:
Yeah, maybe we could benchmark it somehow -- like `touch */src/lib.rs && cargo build`, and see how big the difference is.